popUp.css

```diff
@@ -0,0 +1,212 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;500;700&display=swap");
+
+.project-detail-card {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: #fff;
+  overflow-x: hidden;
+}
+
+.modal-content {
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+
+  /* overflow-y: scroll; */
+  margin: 0 auto;
+  height: 100%;
+  width: 80%;
+
+  /* height: 1163px;
+  width: 1156px;
+  left: 0;
+  top: 0;
+  border-radius: 16px; */
+}
+
+.modal-prof {
+  width: 100%;
+  color: #7a869a;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 15px;
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+}
+
+.modal-prof .project-liste {
+  padding: 0;
+  list-style-type: circle;
+}
+
+.lister {
+  display: inline-block;
+  list-style-type: circle;
+  margin-right: 20px;
+}
+
+#close-popUp {
+  position: absolute;
+  right: 0;
+  top: 10px;
+  padding: 0 8px;
+  font-size: 35px;
+  cursor: pointer;
+  color: #1a2236;
+}
+
+.modal-image {
+  width: 100%;
+  height: 100%;
+}
+
+.modal-image img {
+  width: 100%;
+  height: 586px;
+}
+
+.modal-head {
+  position: relative;
+  display: flex;
+  width: 100%;
+  background-color: #fff;
+}
+
+.modal-head h2 {
+  width: 50%;
+}
+
+.modal-main {
```

Comment:
Kindly follow the Figma template and the right amount of padding/margin as the previous reviewer requested. Check the images below to see the difference.


Your design
    <img width="1440" alt="Screenshot 2023-01-07 at 3 05 45 PM" src="https://user-images.githubusercontent.com/25887253/211150197-900092f1-ca17-436f-a816-c8d2b9b11b5a.png">

Figma template
<img width="690" alt="Screenshot 2023-01-07 at 3 13 34 PM" src="https://user-images.githubusercontent.com/25887253/211150204-ac25c5a0-a1e9-4428-bd5b-41202d4449ad.png">